IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -246,6 +245,15 @@ - (void)executeRequestImpl:(MSIDRequestCompletionBlock)completionBlock
             }
         }
     }
+    
+    if ([self.delegate skipCachedRefreshToken])
```

Comment:
Can this be a flag on the request rather than delegate?

Reply:
No strong preference here, will update :)

Reply:
Updated